assets/js/googlesitekit-dashboard.js

```diff
@@ -61,10 +56,12 @@ domReady( () => {
 		'js-googlesitekit-dashboard'
 	);
 
+	const { setupModuleSlug } = renderTarget.dataset;
```

Comment:
Let's move this into the condition below to ensure the target exists before accessing it.

Reply:
https://github.com/google/site-kit-wp/pull/3805/commits/182eda9e2a1b8d3e15712fbe6962c05fa5ed234b